GX-TimeFrequency/tests/test_minimax/output_32_freq_points.dat

```diff
@@ -0,0 +1,34 @@
+ Minimax Grid
```

Comment:
I assume this is no longer needed as time and frequency are written to the same file?